WMF Framework/Theme.swift

```diff
@@ -131,18 +132,382 @@ public extension UIColor {
+        descriptionWarning: .orange50,
+        inputAccessoryBackground: .base100,
+        inputAccessoryButtonTint: .base20,
+        pageIndicator: .accent90,
+        pageIndicatorCurrent: .accent50,
+        unreadIndicator: .green50,
+        depthMarker: .base70,
+        diffTextAdd: .base10,
+        diffTextDelete: .base10,
+        diffHighlightAdd: .green90,
+        diffHighlightDelete: .red90,
+        diffStrikethroughColor: .base0,
+        diffContextItemBackground: .lightBorder,
+        diffContextItemBorder: .darkBase90,
+        diffMoveParagraphBackground: .lightBorder,
+        diffCompareAccent: .orange50,
+        diffCompareChangeHeading: .base100)
+    
+    fileprivate static let sepia = Colors(
+        baseBackground: .sepiaBase85,
+        midBackground: .sepiaBase90,
+        paperBackground: .sepiaBase100,
+        chromeBackground: .sepiaBase100,
+        popoverBackground: .base100,
+        subCellBackground: .sepiaBase90,
+        overlayBackground: UIColor.sepiaGray40.withAlphaComponent(0.6),
+        batchSelectionBackground: .accent90,
+        referenceHighlightBackground: .clear,
+        hintBackground: .accent90,
+        hintWarningBackground: .sepiaBase85,
+        animationBackground: .paleNavy,
+        overlayText: .base20,
+        searchFieldBackground: .darkSearchFieldBackground,
+        keyboardBarSearchFieldBackground: .base80,
+        primaryText: .base10,
+        secondaryText: .sepiaGray40,
+        tertiaryText: .sepiaGray40,
+        disabledText: .base80,
+        disabledLink: .sepiaGray40,
+        chromeText: .base10,
+        link: .accent50,
+        accent: .green50,
+        border: .sepiaGray80,
+        shadow: .sepiaGray80,
+        chromeShadow: .base20,
+        cardBackground: .sepiaBase90,
+        selectedCardBackground: .sepiaBase85,
+        cardBorder: .sepiaBorder,
+        cardShadow: .clear,
+        cardButtonBackground: .sepiaBase85,
+        secondaryAction: .accent30,
+        icon: .sepiaGray40,
+        iconBackground: .sepiaBase85,
+        destructive: .red30,
+        error: .red30,
+        warning: .orange50,
+        unselected: .sepiaGray40,
+        blurEffectStyle: .extraLight,
+        blurEffectBackground: .clear,
+        tagText: .base100,
+        tagBackground: .accent70,
+        tagSelectedBackground: .accent50,
+        rankGradientStart: .accent50,
+        rankGradientEnd: .accent50,
+        distanceBorder: .sepiaGray40,
+        descriptionBackground: .orange50,
+        descriptionWarning: .orange50,
+        inputAccessoryBackground: .sepiaBase90,
+        inputAccessoryButtonTint: .base20,
+        pageIndicator: .accent90,
+        pageIndicatorCurrent: .accent50,
+        unreadIndicator: .green50,
+        depthMarker: .sepiaGray40,
+        diffTextAdd: .green50,
+        diffTextDelete: .red30,
+        diffHighlightAdd: nil,
+        diffHighlightDelete: nil,
+        diffStrikethroughColor: .red30,
```

Comment:
Is this lighter red color intentional?

Reply:
This strikethrough red is currently `.red30` which is the same as the other red text in sepia but might need to switch back to `.red75` pending Carolyn's answer to [this question](https://github.com/wikimedia/wikipedia-ios/pull/3592#issuecomment-642869373)